src/components/Utils/Utils.js

```diff
@@ -2,14 +2,17 @@ import { library } from '@fortawesome/fontawesome-svg-core';
 import { far } from '@fortawesome/free-regular-svg-icons';
 import { fab } from '@fortawesome/free-brands-svg-icons';
 import {
-  faCog,
-  faHeart
+  faEnvelope,
+  faGlobe,
+  faHeart,
+  faLocationArrow,
+  faPhone
 } from '@fortawesome/free-solid-svg-icons';
```

Comment:
I use [FontAwesome](https://fontawesome.com/icons) for the icons, which should be free for small projects like ours.